firebase-dataconnect/src/main/kotlin/com/google/firebase/dataconnect/core/DataConnectCredentialsTokenManager.kt

```diff
@@ -395,7 +400,12 @@ internal sealed class DataConnectCredentialsTokenManager<T : Any, L : Any>(
 
       val accessToken = sequencedResult!!.ref.getOrThrow().token
       logger.debug {
-        "$invocationId getToken() returns retrieved token: ${accessToken?.toScrubbedAccessToken()}"
+        "$invocationId getToken() returns retrieved token: " +
+          if (accessToken == PLACEHOLDER_APP_CHECK_TOKEN) {
+            "$accessToken (the \"placeholder\" AppCheck token)"
+          } else {
+            accessToken?.toScrubbedAccessToken()
```

Comment:
Good eye! I think you might be comparing the code in this PR to HEAD, where I committed https://github.com/firebase/firebase-android-sdk/commit/75d5c7e4aa0c52f035e6ea219c1989449499a8e0 to move the "placeholder" logic into `toScrubbedAccessToken()` as a quick follow-up to this PR. I didn't bother sending it out for review since it was a small PR. But thanks for the thorough review.